src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage() {
+        super();
+    }
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "userInput")
+    private MobileElement userInput;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "passwordInput")
+    private MobileElement passwordInput;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "logInButton")
+    private MobileElement btnlogInButton;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.widget.TextView[@text='Forgot the password?']")
```

Comment:
Take the text "Forgot the password?" to a constant.

Reply:
Ready